api/helpers/sdtd/validate-item-name.js

```diff
@@ -42,37 +44,24 @@ module.exports = {
         let server = await SdtdServer.findOne(inputs.serverId);
 
         if (_.isUndefined(server)) {
-            return exits.success(false);
+            return exits.error(new Error("Invalid server"));
         }
 
-        sevenDays.listItems({
-            ip: server.ip,
-            port: server.webPort,
-            authName: server.authName,
-            authToken: server.authToken,
-            itemToSearch: inputs.item
-        }).exec({
-            success: (response) => {
-                if (!response) {
-                    return exits.success(false)
-                }
-                let foundItem = false
-
-                response.map(itemName => {
-
-                    if (itemName === inputs.itemName) {
-                        foundItem = true
-                    }
-                })
-
-                return exits.success(foundItem)
-            },
-            error: (error) => {
-                return exits.success(false)
-            }
-        });
+        try {
+            const itemsFound = (await sails.helpers.sdtdApi.executeConsoleCommand(server, `listitems ${inputs.itemName}`))
+                .result
+                .split('\n')
+                .map(_ => _.trim());
```

Comment:
Convention is usually _ is a throw away variable. This absolutely works but its better to be descriptive
```suggestion
                 .map(itemName => itemName.trim());
```